svthalia/Reaxit
# Add api request logging in debug mode

Closes #418.

### Summary
In debug mode for all API requests the url and response code, or exception will be logged to the console.

### How to test
Steps to test the changes you made:
1. Start app in debug mode.
2. See all request logged in the console.
